Lib/glyphsLib/builder/constants.py

```diff
@@ -210,14 +210,14 @@
 # mode for the ufo2ft KernFeatureWriter whenever the GSFont contain a non-automatic
 # 'kern' feature.
 # See https://glyphsapp.com/tutorials/contextual-kerning
-# NOTE: Even though we use the default "skip" mode for the MarkFeatureWriter,
-# we still must include it this custom featureWriters list, as this is used
-# instead of the default ufo2ft list of feature writers.
-# This also means that if ufo2ft adds new writers to that default list, we
-# would need to update this accordingly... :-/
 DEFAULT_FEATURE_WRITERS = [
     {"class": "KernFeatureWriter", "options": {"mode": "append"}},
-    {"class": "MarkFeatureWriter", "options": {"mode": "skip"}},
+    {
+        "module": "glyphsLib.featureWriters.markFeatureWriter",
+        "class": "ContextualMarkFeatureWriter",
+        "options": {"mode": "skip"},
+    },
+    {"class": "CursFeatureWriter"},
```

Comment:
there is also a GdefFeatureWriter in the ufo2ft's list of default feature writers here:

https://github.com/googlefonts/ufo2ft/blob/9dacc83b6ec9de09bcc06c5e7402f1a179e58b7a/Lib/ufo2ft/featureCompiler.py#L169-L174

If you omit it from the list of filter overrides in the UFO lib, then it will not be run for .glyphs sources any more.
I think glyphsLib no longer writes out a GDEF table in the features.fea, but it sets the public.openTypeCategories in the ufo.lib with the intention of having ufo2ft GdefFeatureWriter use those to build the GDEF.